File: 1894-merge-strings-alternately/merge-strings-alternately.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function (word1, word2) {
    maxLength = Math.max(word1.length, word2.length)
    var result = ""
    for (let i = 0; i < maxLength; i++) {
        if(word1[i]){
            result += word1[i]
        }
        if(word2[i]){
            result += word2[i]
        }
    }

    return result

};

File: 1894-merge-strings-alternately/merge-strings-alternately.go
func mergeAlternately(word1 string, word2 string) string {
	var maxLength = 0

    var newWord1 []string
    var newWord2 []string

    newWord1 = strings.Split(word1,"")
    newWord2 = strings.Split(word2,"")

	if len(newWord1) > len(newWord2) {
		maxLength = len(newWord1)
	} else {
		maxLength = len(newWord2)
	}

	result := ""

	for i := 0; i < maxLength; i++ {
		if len(newWord1) > i {
			result += newWord1[i]
		}
		if len(newWord2) > i {
			result += newWord2[i]
		}
	}

	fmt.Println("Hello ", result)

	return result
}